.github/workflows/manpage.yml

```diff
@@ -0,0 +1,29 @@
+name: Building man page and man.test
+on:
+  push:
+    paths:
+      - 'docs/**'
+      - '.github/workflows/manpage.yml'
```

Comment:
Add the target files here to make WF fail on unexpected edits (e.x. add a test to man.test but not to manual.yml).